src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,36 @@
-import React from 'react';
+import React, {
+  ReactEventHandler, useCallback, useEffect, useState,
+} from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  selectedTodo: Todo
+  onClose: ReactEventHandler
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, onClose }) => {
+  const [isLoadedUser, setIsLoadedUser] = useState(false);
```

Comment:
R U sure you need this state?
i mean you could evaluate the `user` - since initially it's `null` (hence false equivalent) - and display loader like `!user ? <Loader />` 
instead of `!isLoadedUser ? <Loader />`